keyboards/rainkeebs/trailmix/rules.mk

```diff
@@ -0,0 +1,4 @@
+# Build Options
+#   change yes to no to disable
+#
+ENCODER_MAP_ENABLE = yes
```

Comment:
```suggestion
# This file intentionally left blank
```

Reply:
I only know realise that I actually had to have the `ENCODER_MAP_ENABLE  = yes` in the `rules.mk` inside the `<keymap>` directory. Silly me!
Sorry for the back and forth about that value.